designs/dragonphy_top/constraints/gen_constraints.py

```diff
@@ -144,22 +144,6 @@
 # IOs for output buffer are all false paths
 set_false_path -through [get_pins -of_objects idcore/out_buff_i]
 
-###############
```

Comment:
we no longer need this because the mux settings only affect the behavior of `analog_core`, which is a black box for top-level synthesis